server/routes/user.js

```diff
@@ -254,4 +284,26 @@ router.post("/update", function (req, res, next) {
   });
 });
 
+/**
+ * update user role
+ */
+router.post("/updaterole", function (req, res, next) {
```

Comment:
The convention we've been using for route names is words separated by hyphens. So "update-role" instead of updaterole.